products/statement-generator/src/components/EditContent.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { makeStyles, createStyles } from '@material-ui/core';
+
+import FormStateContext from 'contexts/FormStateContext';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      display: 'flex',
+      height: '100%',
+      width: '70%',
+    },
+    form: {
+      height: '120px',
+      width: '100%',
+      display: 'flex',
+      flexDirection: 'column',
+      margin: '0 auto',
+    },
```

Comment:
Instead of the `root` styles you created here  try using `primaryContainer` from the `utilityStyles`. Same with replacing `form` with the `contentContainer` styles. Similar to here https://github.com/hackforla/expunge-assist/blob/master/products/statement-generator/src/pages/PreviewPage.tsx#L86-L87